sklearn/utils/sparsefuncs.py

```diff
@@ -623,6 +623,10 @@ def count_nonzero(X, axis=None, sample_weight=None):
         axis = 1
     elif axis == -2:
         axis = 0
+    elif X.format == "csc":
```

Comment:
So you are making count_nonzero support CSC format, which seems fine, can you update the docstring, since it currently says "it should be of CSR format"

Reply:
Thanks for these comments -- I've updated the PR accordingly.